client/state/reader/related-sites/actions.js

```diff
@@ -0,0 +1,19 @@
+import {
+	READER_RELATED_SITES_REQUEST,
+	READER_RELATED_SITES_RECEIVE,
+} from 'calypso/state/reader/action-types';
+
+import 'calypso/state/data-layer/wpcom/read/related/sites';
+
+import 'calypso/state/reader/init';
+
+export const requestRelatedSites = ( { offset = 0, number = 4, tag = '' } ) => ( {
```

Comment:
In the future, if we want to make the `/tag/$tag` page work with Server Side Rendering, we would have to move the data fetching into the controller like we do with the `/tags` page e.g. https://github.com/Automattic/wp-calypso/blob/trunk/client/reader/tags/controller.tsx#L54

I think it's fine for now though, it matches what is currently there.